code/game/objects/items/devices/radio/radio.dm

```diff
@@ -225,7 +225,7 @@
 		spans = list(M.speech_span)
 	if(!language)
 		language = M.get_selected_language()
-	SEND_SIGNAL(src, COMSIG_RADIO_MESSAGE, M, message, channel)
+	SEND_SIGNAL(src, COMSIG_RADIO_MESSAGE, M, message, channel, message_mods)
```

Comment:
Document that message mods are being sent to the signal now in the place that the signal is defined.
Update instances that register this signal so that they have the correct parameters, even if they don't use the message mods parameter.

Reply:
> Document that message mods are being sent to the signal now in the place that the signal is defined. Update instances that register this signal so that they have the correct parameters, even if they don't use the message mods parameter.

there's only a single proc using this signal and that's it.

Reply:
The actual file that contains the definition for COMSIG_RADIO_MESSAGE will have a comment attached to it saying how it should be registered. That needs updating to include the additional parameters being sent.

Reply:
There's no document about it so that I am not really sure what I am supposed to do

Reply:
Search for #define and then the signal name. All signals should have a comment next to them which says where the signal originates from and then a list of params `//! Called from base of atom/move (atom/oldLoc, forced)`

Reply:
Pushed. I hope it's what I am supposed to do.